content/content-tool/tool/src/java/org/sakaiproject/content/exception/ZipMaxTotalSizeException.java

```diff
@@ -0,0 +1,11 @@
+package org.sakaiproject.content.exception;
+
+import org.sakaiproject.exception.SakaiException;
+
+/**
+ * <p>
+ * ZipMaxTotalSizeException is thrown whenever the size of all files to be zipped exceeds the maximum permitted in default.sakai.properties.
+ * </p>
+ */
+public class ZipMaxTotalSizeException extends SakaiException {
```

Comment:
Comment for future: I would have been tempted to embed the size of the limit in the exception. It then makes it easier when displaying a message to the user to be able to say what the limit it.

Reply:
Good point.